src/pynxtools/nomad/schema.py

```diff
@@ -241,9 +272,25 @@ def __get_documentation_url(
         if xml_node is None:
             break
 
+    definitions_url = get_definitions_url()
+
+    doc_base = __NX_DOC_BASES.get(
+        definitions_url, "https://manual.nexusformat.org/classes"
+    )
     nx_package = xml_parent.get("nxdl_base").split("/")[-1]
     anchor = "-".join([name.lower() for name in reversed(anchor_segments)])
-    return f"{__NX_DOC_BASE}/{nx_package}/{anchor_segments[-1]}.html#{anchor}"
+    return f"{doc_base}/{nx_package}/{anchor_segments[-1]}.html#{anchor}"
+
+
+def __rename_nx_to_nomad(name: str) -> str:
+    """
+    Rename the NXDL name to NOMAD.
+    For example: NXobject -> NOMADObject
+    """
+    if name.startswith("NX"):
+        # ToDo: replace by a useful prefix
+        return name.replace("NX", "NX")
```

Comment:
For the release, we could use NOMAD, since these are not anymore just NX defnitions mapped to metainfo, but became actual BaseSections. If the prefix NOMAD is too long, we could go for BS.